hugegraph-core/src/main/java/com/baidu/hugegraph/task/TaskManager.java

```diff
@@ -242,17 +237,19 @@ public int pendingTasks() {
     }
 
     protected void notifyNewTask(HugeTask<?> task) {
-        // Notify to schedule tasks initiatively when have new task
-        this.schedulerExecutor.submit(this::scheduleOrExecuteJob);
+        Queue<Runnable> queue = ((ThreadPoolExecutor) this.schedulerExecutor)
+                                                          .getQueue();
+        if (queue.size() <= 1) {
```

Comment:
It may schedule more tasks at the same time, and these tasks will not be processed.

Reply:
it's ok to not notify again if there are more than 1 task in queue(like two, one is timer task, one is immediate task), we don't want too many  immediate tasks to be inserted into queue, one notify will cause all the tasks processed.